.buildkite/scripts/steps/artifacts/build.sh

```diff
@@ -25,7 +25,9 @@ buildkite-agent artifact upload "dependencies-$FULL_VERSION.csv.sha512.txt"
 buildkite-agent artifact upload 'i18n/*.json'
 cd -
 
-cd .beats
-buildkite-agent artifact upload 'metricbeat-*'
-buildkite-agent artifact upload 'filebeat-*'
-cd -
+if [[ -n "$(ls -A .beats 2>/dev/null)"]]; then
```

Comment:
`-d` checks if a path is a directory
```suggestion
if [ -d .beats ]; then
```